### PR TITLE
Quick Actions: when no actions present don't render menu

### LINKED ISSUE #10950: Quick Actions: Empty when multiple elements selected
## Bug Description

When multiple elements which are of different types are selected, the quick actions menu is empty, but we are still showing the empty container.

## Expected Behaviour

The quick actions menu shouldn't show if there are no actions for the selected elements.
## Steps to Reproduce
Add two elements of different types, eg. a text element and a shape, or a an image and a video.
Select them both
Notice quick actions is an empty box. 

## Additional Context

- Plugin Version:
- WordPress Version:
- Operating System:
- Browser:

### PR DESCRIPTION
**Pull request by BrookeGraham**
## Context

If you select two differing element types, the quick actions context menu renders an empty box. Let's not do that. 

## Relevant Technical Choices
Checks if there are any quick actions, if not, don't render the context menu at all .
Added a `data-testid` since I wasn't sure how else to check for `null` on this context menu. 🤔 

## User-facing changes
|🙃|🙂|
|--|--|
|Screen Shot 2022-03-23 at 3 58 49 PM|Screen Shot 2022-03-23 at 3 56 01 PM|

## Testing Instructions

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. add two different element types to a page and selected them both.
2. notice no empty box were quick actions should be

### Does this PR have a security-related impact?
no

### Does this PR change what data or activity we track or use?
no

### Does this PR have a legal-related impact?
no

## Checklist
no

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/googleforcreators/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/googleforcreators/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #10950

### CODE REVIEW

File: packages/story-editor/src/components/canvas/karma/quickActions.karma.js [hunk 1]
```diff
@@ -56,6 +56,60 @@ describe('Quick Actions integration', () => {
...
+
+      await fixture.act(() =>
+        insertElement('text', {
+          font: TEXT_ELEMENT_DEFAULT_FONT,
+          content: 'Hello world!',
+          x: 10,
+          y: 20,
+          width: 400,
+        })
+      );
```
Comment (samwhale) [inline packages/story-editor/src/components/canvas/karma/quickActions.karma.js]: 💭  Miina asked me a little while back to limit our use of `insertElement` and instead add elements using the library.

Let's replace these with adding an element directly from the library: https://github.com/GoogleForCreators/web-stories-wp/blob/b3a2b97493ecb9f116e7c7015d69ee3b05de8927/packages/story-editor/src/components/floatingMenu/elements/karma/more.karma.js#L43-L94

File: packages/story-editor/src/components/canvas/pageSideMenu.js [hunk 2]
```diff
@@ -77,69 +77,77 @@ function PageSideMenu() {
...
-            );
+      {
+        // Dont render a menu wrapper if there are no quick actions
+        quickActions.length ? (
+          <ContextMenu
+            isInline
+            isAlwaysVisible
+            isIconMenu
+            disableControlledTabNavigation
+            data-testid={'Element quick actions'}
```
Comment (samwhale) [inline packages/story-editor/src/components/canvas/pageSideMenu.js]: Don't need string interpolation here
```suggestion
            data-testid="Element quick actions"
```

File: packages/story-editor/src/components/canvas/karma/quickActions.karma.js [hunk 3]
```diff
@@ -56,6 +56,60 @@ describe('Quick Actions integration', () => {
...
+
+      expect(
+        fixture.screen.queryByTestId('Element quick actions')
+      ).not.toBeNull();
+
+      await fixture.events.keyboard.down('Shift');
+      await clickOnTarget(
+        fixture.editor.canvas.framesLayer.frame(shapeElement.id).node
+      );
+      await fixture.events.keyboard.up('Shift');
```
Comment (samwhale) [inline packages/story-editor/src/components/canvas/karma/quickActions.karma.js]: 💭  Would be good to verify that the quick action menu is visible prior to selecting the second element.

**Review by samwhale — APPROVED** [review]
Love the test! Just left one little comment. Works for me!

**Review by maxyinger — APPROVED** [review]

File: packages/story-editor/src/components/canvas/karma/quickActions.karma.js (hunk 3 above)
Comment (BrookeGraham) [reply packages/story-editor/src/components/canvas/karma/quickActions.karma.js]: I am checking for it. Lines 101-103

File: packages/story-editor/src/components/canvas/karma/quickActions.karma.js (hunk 1 above)
Comment (BrookeGraham) [reply packages/story-editor/src/components/canvas/karma/quickActions.karma.js]: You caught me, I was being lazy.

File: packages/story-editor/src/components/canvas/karma/quickActions.karma.js (hunk 3 above)
Comment (samwhale) [reply packages/story-editor/src/components/canvas/karma/quickActions.karma.js]: Oh great! Sorry, I missed it

**Review by BrittanyIRL — APPROVED** [review]

**Pull request merged**